nodewatcher/nodewatcher.py

```diff
@@ -149,14 +158,15 @@ def stackReady(stack_name, region, proxy_config):
     stacks = cfn_client.describe_stacks(StackName=stack_name)
     return stacks['Stacks'][0]['StackStatus'] in ['CREATE_COMPLETE', 'UPDATE_COMPLETE', 'UPDATE_ROLLBACK_COMPLETE']
 
+
 def main():
     logging.basicConfig(
         level=logging.INFO,
         format='%(asctime)s %(levelname)s [%(module)s:%(funcName)s] %(message)s'
     )
-    log.info("nodewatcher startup")
     instance_id = getInstanceId()
     hostname = getHostname()
+    log.info('nodewatcher startup for instance %s with hostname %s' % (instance_id, hostname))
```

Comment:
I'd split this log in two different logs:
```
log.info("nodewatcher startup")
instance_id = ...
hostname = ...
log.info('Instance = %s, hostname = %s' % (instance_id, hostname))
```
This because the _startup_ log is useful to understand when the nodewatcher has been restarted and if the `getInstanceId` or the `getHostname` fail we don't have this information.